app/assets/javascripts/discourse/app/components/user-card-contents.hbs

```diff
@@ -27,54 +27,48 @@
       </div>
     {{else}}
       <div class="card-row first-row">
-        <div class="user-card-avatar">
-          {{#if this.contentHidden}}
-            <span class="card-huge-avatar">{{bound-avatar
-                this.user
-                "huge"
-              }}</span>
-          {{else}}
-            <a
-              {{on "click" this.handleShowUser}}
-              href={{this.user.path}}
-              class="card-huge-avatar"
-            >{{bound-avatar this.user "huge"}}</a>
-          {{/if}}
-
-          <UserAvatarFlair @user={{this.user}} />
-
-          <div>
-            <PluginOutlet
-              @name="user-card-avatar-flair"
-              @connectorTagName="div"
-              @outletArgs={{hash user=this.user}}
-            />
-          </div>
-        </div>
-        <div class="names">
-          <div
-            class="names__primary
-              {{this.staff}}
-              {{this.newUser}}
-              {{if this.nameFirst 'full-name' 'username'}}"
-          >
+        <PluginOutlet
+          @name="user-card-main-info"
+          @outletArgs={{hash
+            user=this.user
+            post=this.post
+            contentHidden=this.contentHidden
+            handleShowUser=this.handleShowUser
+          }}
+        >
+          <div class="user-card-avatar" aria-hidden="true">
```

Comment:
This looks like an HTML change which may affect themes/plugins. Is it essential for the backport?

Reply:
it is not, I'm removing it!

Reply:
@branquinhoaa this one is still outstanding. Is it droppable?